config.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY', '')
+
```

Comment:
Need to add the VERSION with relay scheme version, like in google safe browsing

Reply:
Done